newrelic-php-agent/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
@cw-niinuma 
For `appVersion`, specify the support version of NewRelic's PHP Agent.